### ISSUE TITLE
New Survey: Developer Experience

### ISSUE THREAD

**Issue by reese-lee** [open-telemetry/sig-end-user#76]
Opening this issue per @tsloughter's request in [this Slack convo](https://cloud-native.slack.com/archives/C01RT3MSWGZ/p1729678610730479). They have requested this survey to run after the Docs survey closes. 

Checklist:
- [x] Create survey -- sounds like this is in-progress
- [x] Promote survey on socials
- [x] Add link to survey on opentelemetry.io banner
- [x] Summarize survey findings (blog post on opentelemetry.io)
- [ ] Schedule end user panel for DevEx
- [ ] Present findings to maintainers
- [ ] Present findings to DevEx group

**Comment by avillela** [comment]
Link to survey questions [here](https://docs.google.com/document/d/1JSl87hJVoOGigSFyTMybZr2KoFvGdo4BsdOepWu9BPM/edit?tab=t.0)

**Comment by reese-lee** [comment]
I've opened a PR to add a banner announcement for the survey; I'll mark it as done once the PR has been approved and merged.

**Comment by reese-lee** [comment]
PR merged, banner up!

**Comment by danielgblanco** [comment]
I set the form to stop receiving responses. We should coordinate next steps.

**Comment by tsloughter** [comment]
@danielgblanco We are bringing together summaries and working on a blog post. Is there anything else we should be working on in the next steps? Thanks!

**Comment by danielgblanco** [comment]
I don't think so! Thanks! I just noticed that the survey had remained open for a bit longer than expected. I disabled the "accept new responses" setting.

**Comment by reese-lee** [comment]
* Blog post has been published: https://opentelemetry.io/blog/2025/devex-survey/.  
* Following up on some of the remaining items: https://cloud-native.slack.com/archives/C01S42U83B2/p1744410008273259.

**Comment by AndrejKiri** [comment]
@reese-lee FYI @julianocosta89 [asked on Slack](https://cloud-native.slack.com/archives/C01RT3MSWGZ/p1744189599483249) for guidance on contributing the DevEx survey data, so I think he plans to do that as well.

**Comment by julianocosta89** [comment]
@reese-lee I think we are good to close this issue.
